src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml

```diff
@@ -23,33 +23,17 @@
 </tr>
 @if (Model.Diagnostics.Any())
 {
+    var errorDiags = Model.Diagnostics.Where(d => d.Level == 0 || d.Level == APIView.CodeDiagnosticLevel.Error);
```

Comment:
Add a Default member to CodeDiagnosticLevel with value 0

Reply:
Done